hack/make/.build-rpm/docker-core.spec

```diff
@@ -0,0 +1,104 @@
+Name: docker-core
+Version: %{_rpmversion}
+Release: 1%{?dist}
+Summary: The open-source application container engine
+
+License: ASL 2.0
+Source: docker-core.tar.gz
+
+URL: https://dockerproject.com
+Vendor: Docker
+Packager: Docker <support@docker.com>
+
+BuildRequires: systemd
+Requires: /bin/sh
+Requires: libc.so.6
+Requires: libdevmapper.so.1.02
+Requires: libpthread.so.0
+Requires: libsqlite3.so.0
+Requires: iptables
+
+Conflicts: docker
+Conflicts: docker-io
+Conflicts: lxc-docker
+Conflicts: lxc-docker-virtual-package
+
+%description
+Docker is an open source project to pack, ship and run any application as a
+lightweight container
+ 
+Docker containers are both hardware-agnostic and platform-agnostic. This means
+they can run anywhere, from your laptop to the largest EC2 compute instance and
+they can run anywhere, from your laptop to the largest EC2 compute instance and
+everything in between - and they don't require you to use a particular
+language, framework or packaging system. That makes them great building blocks
+for deploying and scaling web apps, databases, and backend services without
+depending on a particular stack or provider.
+
+%prep
+%autosetup -n %{name}
+
+%build
+./hack/make.sh dynbinary
+# ./docs/man/md2man-all.sh runs outside the build container (if at all), since we don't have go-md2man here
+
+%check
+./bundles/%{_version}/dynbinary/docker -v
+
+%install
+# install docker
+mkdir -p $RPM_BUILD_ROOT/%{_bindir}
+cp -aT bundles/%{_version}/dynbinary/docker-%{_version} $RPM_BUILD_ROOT/%{_bindir}/docker
+mkdir -p $RPM_BUILD_ROOT/usr/libexec/docker
+cp -aT bundles/%{_version}/dynbinary/dockerinit-%{_version} $RPM_BUILD_ROOT/usr/libexec/docker/dockerinit
+
+# copy udev rules
+mkdir -p $RPM_BUILD_ROOT/etc/udev/rules.d
+cp -aT contrib/udev/80-docker.rules $RPM_BUILD_ROOT/etc/udev/rules.d/80-docker.rules
+
+# add init scripts
+mkdir -p $RPM_BUILD_ROOT/%{_unitdir}
+cp -aT contrib/init/systemd/docker.service $RPM_BUILD_ROOT/%{_unitdir}/docker.service
+cp -aT contrib/init/systemd/docker.socket $RPM_BUILD_ROOT/%{_unitdir}/docker.socket
+
+# add bash completions
+mkdir -p $RPM_BUILD_ROOT/usr/share/bash-completion/completions
+mkdir -p $RPM_BUILD_ROOT/usr/share/zsh/vendor-completions
+mkdir -p $RPM_BUILD_ROOT/etc/fish
+cp -aT contrib/completion/bash/docker $RPM_BUILD_ROOT/usr/share/bash-completion/completions/docker
+cp -aT contrib/completion/zsh/_docker $RPM_BUILD_ROOT/usr/share/zsh/vendor-completions/_docker
+cp -aT contrib/completion/fish/docker.fish $RPM_BUILD_ROOT/etc/fish/docker.fish
```

Comment:
The old deb installer puts them in "/etc/fish/completions/", but the new
one punts completely on doing anything with fish ATM.